pages/windows/reg-delete.md

```diff
@@ -0,0 +1,19 @@
+# reg delete
+
+> Delete keys or their values from the registry.
+
+- Delete a specific registry key:
+
+`reg delete {{key_name}}`
+
+- Delete a value under a specific key:
+
+`reg delete {{key_name}} /v {{value}}`
+
+- Delete all values recursively under the specified key:
+
+`reg delete {{key_name}} /va`
+
+- Forcefully deletes the key or value without a prompt:
```

Comment:
Hmm .. you mention "key or value". I thought values were data under a key. So will this be the same as `/va` ? But if you want to delete a particular value, don't you need to specify the parent key also ?

Reply:
It's mainly to be combined with any of the other commands above.

So to delete a value, you'd combine it with `/v {{value}}`

Reply:
Understood. But the description does not match with the example here, you mention "key or value" but the example just has key_name. Maybe it's better if you do a `/f /va` and mention that `Forcefully delete all values recursively under a key`. That would indicate that `/f` can be combined with `/v {{value}}` too.